api/initialize.py

```diff
@@ -1,3 +1,55 @@
-from app.models import db
+from app.models import db, User, Election, Question
+from app.config import TEST_SERVER, MEMBERSHIP_API, SEED_DATA, TOKEN
+import json
+import requests
+import datetime
 
-db.create_all()
\ No newline at end of file
+db.drop_all()
+db.create_all()
+
+if(SEED_DATA):
+
+    User.query.delete()
+    db.session.commit()
+    print(TOKEN)
+    r = requests.get(MEMBERSHIP_API + 'api/v1/leaderboard', headers = TOKEN)
+
+    print(r.status_code)
+    print(r.json())
+
+    leaderboard_user = r.json()
+
+    users = leaderboard_user['leaderboard']
+
+    for u in users:
+        uName = u['firstName'] + " " + u['lastName']
+        
+        board = False #True if u['accessType'] == 'STAFF' else
+
+        newUser = User(userName=uName, uuid="null", canVote=[], boardMember=board)
+        db.session.add(newUser)
+    db.session.commit()
+
+    if(TEST_SERVER):
+        Election.query.delete()
+        Question.query.delete()
+        db.session.commit()
+
+
+        q1 = Question(question="q1", votes='{ "q1answer1":0, "q1answer2":0 }', voteType="SPTP")
+        q2 = Question(question="q2", votes='{ "q2answer1":0, "q2answer2":0 }', voteType="Proportional")
+        q3 = Question(question="q2", votes='{ "q2answer1":0, "q2answer2":0 }', voteType="SPTP")
+
+        db.session.add_all([q1,q2,q3])
+        db.session.commit()
+
+
+        q1ID = Question.query.filter_by(question="q1").first().id
+        q2ID = Question.query.filter_by(question="q2").first().id
```

Comment:
Similar to the above here? Still `question="q2"` instead of `question=q3`.
```suggestion
        q3ID = Question.query.filter_by(question="q3").first().id
```